ContentSerializer/ProgramInterface.cs

```diff
@@ -549,6 +551,45 @@ public int ExportToMap(string target)
             return 0;
         }
 
+        public void FindFiles(string section, string name)
+        {
+            UInt32 hash = HashFunctions.GetInibinHash(section, name);
+            foreach(var entry in _files)
+            {
+                var file = _manager.ReadFile(entry.FullName).Uncompress();
+                var inibin = Inibin.DeserializeInibin(file, entry.FullName);
+                foreach (var kvp in inibin.Content)
+                {
+                    if(kvp.Key == hash)
+                        Console.WriteLine("{0}: {1}", entry.FullName, kvp.Value.Value.ToString());
+                }
+            }
+        }
+
+        public void FindValueType(int type,bool filter)
+        {
+            foreach(var entry in _files)
+            {
+                var file = _manager.ReadFile(entry.FullName).Uncompress();
+                var inibin = Inibin.DeserializeInibin(file, entry.FullName);
+                foreach (var kvp in inibin.Content)
+                {
+  
+                    if(kvp.Value.Type == type)
+                    {
+                        if (_draft.Hashes.ContainsKey(kvp.Key))
+                        {
+                            var sn = _draft.Hashes[kvp.Key].First();
+                            Console.WriteLine("{0} {2} {3}: {1}", entry.FullName, kvp.Value.Value.ToString(),
+                                sn.Key, sn.Value.First());
+                        }else if (!filter) { 
```

Comment:
Move the `else if (!filter)` to a new line please, or at the very least the opening bracket.